visualize_service:
  timer_duration: 0.01